vendor/modules.txt

```diff
@@ -10,6 +10,8 @@ github.com/google/go-dap
 github.com/hashicorp/golang-lru/simplelru
 # github.com/inconshreveable/mousetrap v1.0.0
 github.com/inconshreveable/mousetrap
+# github.com/intel-go/cpuid v0.0.0-20200819041909-2aa72927c3e2
```

Comment:
You should revert this change and remove the vendor/github.com/intel-go/cpuid directory.